<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RMAddUI</class>
 <widget class="QDialog" name="RMAddUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="4">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>批号：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEdit_name"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>备注：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QSpinBox" name="spinBox_amount">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleEdit" name="lineEdit_conc"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>名称*：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>浓度*：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>单位：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="7">
      <widget class="QLineEdit" name="lineEdit_mark"/>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>规格：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>介质：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>不确定度：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_number"/>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>数量：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>编号*：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>有效期：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLineEdit" name="lineEdit_storePos"/>
     </item>
     <item row="0" column="7">
      <widget class="QLineEdit" name="lineEdit_spec"/>
     </item>
     <item row="1" column="7">
      <widget class="QLineEdit" name="lineEdit_Uncertainty"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_solvent"/>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>保存位置：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>厂商：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="lineEdit_unit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_manufactor"/>
     </item>
     <item row="2" column="3">
      <widget class="QDateEdit" name="dateEdit_validity"/>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="lineEdit_lotNumber"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>说明：带*为必填项目。</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,1">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>824</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btAdd">
       <property name="text">
        <string>增加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btFinished">
       <property name="text">
        <string>完成</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">qdoubleedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
